app/models/measurement.rb

```diff
@@ -1,5 +1,7 @@
 class Measurement < ActiveRecord::Base
   belongs_to :person
+  scope :descending, -> { order(date: :desc) }
+  scope :latest, -> { order(date: :asc).last }
```

Comment:
Anything that doesn't return an association (i.e. when you call `last`, for instance) should be a class method.